customer.go

```diff
@@ -189,7 +189,10 @@ func (cr *CustomerRequest) GetList(customerListInput *CustomerListInput) (*Custo
 	}
 
 	queryParams := make(map[string]string)
-	json.Unmarshal(jsonQueryParams, &queryParams)
+	err = json.Unmarshal(jsonQueryParams, &queryParams)
+	if err != nil {
```

Comment:
Hey 👋 

You could also make this kind of statement to keep your err inside the scope of the if block:

```suggestion
	if err := json.Unmarshal(jsonQueryParams, &queryParams); err != nil { 
	    // Your error code.
	}
```

It can be useful sometimes, but it's mostly some sugar.

Reply:
Yeah I'll make this way thanks!